SampleProjects/DoSomething/README.md

```diff
@@ -3,13 +3,13 @@
 This project is a template for a CI-enabled (and unit testable) Arduino project of your own.
 
 
-### Features
+## Features
```

Comment:
Here again, the linter is doing something incorrect. The `###` title is by design, because it does not produce a horizontal rule when rendered -- an undesirable division within the top section of this document.

Reply:
Here the linter wanted to increment the heading by one instead of two. I updated the linter to ignore this.